docs/examples/nginx/templates/configmap.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   labels:
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
```

Comment:
Before using `AppVersion` here can you add it to the `Chart.yaml` so there will be a value.